app/controllers/RiderStageConnectionController.java

```diff
@@ -48,15 +48,14 @@ public RiderStageConnectionController(StageRepository stageRepository, RiderRepo
         });
     }
 
-
-    public CompletionStage<Result> getStage(int stageId) {
-        return stageRepository.getStage(stageId).thenApplyAsync(stage -> {
-           return ok(stage);
+    public CompletionStage<Result> getRiderStageConnection(long stageId, long riderId) {
+        return riderStageConnectionRepository.getRiderStageConnectionByRiderAndStage(stageId, riderId).thenApplyAsync(riderStageConnection -> {
+            return ok(toJson(riderStageConnection));
         }).exceptionally(ex -> {
             Result res = null;
             switch (ExceptionUtils.getRootCause(ex).getClass().getSimpleName()){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)